package.json

```diff
@@ -94,7 +94,7 @@
     "eslint_d": "^5.2.1",
     "extract-text-webpack-plugin": "^3.0.2",
     "file-loader": "^1.1.5",
-    "flow-bin": "^0.61.0",
+    "flow-bin": "^0.62.0",
```

Comment:
Upgraded Flow when we thought that might fix things. It didn't, but may as well keep the upgrade.